### DISCUSSION TITLE
After creating a new Android project, the example apps don't render maps

### DISCUSSION THREAD

**Discussion by miki151** [mapsforge/mapsforge#1350] [Q&A] [answered]
I had some fun editing the sample Android app, and finally created a new project and moved my code there. Unfortunately this new app won't render any maps. It does render all other non-map layers, like MyLocationOverlay, a Polyline or the grid and coordinates overlays found in DiagnosticsMapViewer.

I also used a ThreadedLabelLayer and this **didn't** render anything. I haven't noticed any errors in the Android logs.

I've created a repository with a downsized project that's basically a copy of the SimplestMapViewer example https://github.com/miki151/offlinemaps. I've spent at least 5 hours trying to figure out why it won't render maps and I don't have any further ideas. I suspect that it has to do with configuration, since the code is the same as in the example. Considering which layers render and which don't, it seems like it's either MapDataStore or some bad parameters like DPI or layouts passed to the MapView.

If anyone could take a look and give me a hint on what I could've missed or what else I could try I would be very grateful. Let me know if I should supply any other information.

**Comment by devemux86** [comment, marked as answer]
See the [GettingStarted](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-samples-android/src/main/java/org/mapsforge/samples/android/GettingStarted.java) sample.

**Comment by miki151** [comment]
I finally figured it out, I forgot to call:

AndroidGraphicFactory.createInstance(this);
